Clase24Herencia/Telefono/telefono.ts

```diff
@@ -0,0 +1,56 @@
+class Telefono {
+    private estaPrendido: boolean;
+    private bateriaActual: number;
+
+    public constructor() {
+        this.estaPrendido = false;
+        this.bateriaActual = 100;
+    }
+
+    public mandarMensaje(mensaje: string): void {
+        this.estaPrendido = true;
+        console.log('Mensaje a Enviar ', mensaje);
+    }
+    public hacerLlamada(numTE: number): void {
+        this.estaPrendido = true;
+        console.log('Llamar al Numero ', numTE);
+    }
+    public prenderApagar(): void {
+        if (this.estaPrendido)
```

Comment:
para simplificar esto podes hacer directamente `this.estaPrendido = !this.estaPrendido` entonces asignas el valor contrario al que tiene la variable

Reply:
Buenísimo! No sabia eso!